gnocchi/storage/_carbonara.py

```diff
@@ -391,8 +388,6 @@ def _compute_and_store_timeseries(self, metric, measures):
             LOG.debug("Skipping %s (already processed)", metric)
             return
 
-        measures = sorted(measures, key=operator.itemgetter(0))
```

Comment:
It's not. I thought I could optimize that but I can't. That's what it bugged. Thanks!